assets/src/web-stories-block/block/globals.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const { webStoriesBlockSettings } = window;
```

Comment:
Good idea to move this to its own file 👍

Reply:
What happens if this variables doesn't exist?

Reply:
We have localized this and sent to the script if the variable is not present when this run that means there something up with the script orders, else we can say this will always exists.